magento/pwa-studio
# [PWA-272] [Feature]: "Edit Item" From Cart Page Kebab Menu

## Description

As a shopper, I want to edit my product from within the cart page workflow in order to change configurable options (size, color, quantity, etc.) prior to purchasing. 

Today, edit exists within the mini-cart experience but during the build out of the new Cart page we realized that reusing this same experience requires additional design. This is a follow up story to  PWA-238.  

Need to decide on design treatment for edit item experience within the full cart. 

## Related Issue





* [[PWA-272](https://jira.corp.magento.com/browse/PWA-272)] [Feature]: "Edit Item" From Cart Page Kebab Menu
## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add Simple and Configurable Product to cart and go to `/cart` page
2. Verify no edit option available for simple product
3. Edit Config Product, verify that all selected options are correct
4. Change some configurable options and quantity and update
5. Verify cart is updated with selections and totals are updated

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
